### DISCUSSION TITLE
https://thanhle.blog/blog/game-dau-tu-dau-tu-vao-ban-than

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#29] [Ideas]
# https://thanhle.blog/blog/game-dau-tu-dau-tu-vao-ban-than

Đầu tư vào bản thân là câu nói huyền thoại của Warren Buffett, vậy cùng xem đầu tư vào bản thân là làm gì nhé

https://thanhle.blog/blog/game-dau-tu-dau-tu-vao-ban-than

**Comment by lamhoangvu217** [comment]
Bài này hay quá anh. Mong anh ra nhiều bài như này hơn nữa ạ 🥶

**Reply by thanhlmm** [reply]
Bạn nhớ subcribe email nha. Mình viết bài mỗi tuần á 😄

**Comment by dfjerry** [comment]
Ơ e bị lạc vào blog của a rồi, đọc cuốn quá =))

**Comment by hai123vn** [comment]
quá hay anh ơi, nhưng mà ra bài mỗi tuần là sai nhé khoảng 2,3 tuần mới ra 1 bài ạ ^^

**Reply by thanhlmm** [reply]
Bái này là năm ngoái rồi, và năm ngoái thì a mỗi tuần 1 bài mà :)))

**Comment by khiem20tc** [comment]
Bài hay thế mà giờ em mới thấy :))) em cũng hay tự hỏi tại sao bản thân phải luôn trong tư thế "chạy" như thế, liệu làm 1 công việc bình bình, sống 1 cuộc đời bình thường cả đời thì có ok không :)
